CHANGELOG.md

```diff
@@ -23,6 +23,8 @@ project adheres to [Semantic Versioning](http://semver.org/).
 
 ### Breaking
 
+- types: fixed type for `registry.getMetricsAsArray()`
```

Comment:
Since this isn't a breaking change, can you move it from here to `### Changed` please?

Reply:
erf yes sorry about that. It's not in the right version either, moving it.